<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Convertor</class>
 <widget class="QWidget" name="Convertor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>138</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Unit Convertor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,10,0,0">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Category:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="m_categoryCombo">
       <item>
        <property name="text">
         <string>Temperature</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Distance</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Currency</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Volume</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mass</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_convert">
       <property name="text">
        <string>convert</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_reverse">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>From:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="m_fromCombo"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLineEdit" name="m_fromLine"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::VLine</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>To</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="m_toCombo"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLineEdit" name="m_toLine"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
